tests/pages/multipleOperatorsServiceList.test.tsx

```diff
@@ -0,0 +1,252 @@
+                                { operatorPublicName: 'Test1', nocCode: 'N1' },
+                                { operatorPublicName: 'Test2', nocCode: 'N2' },
+                                { operatorPublicName: 'Test3', nocCode: 'N3' },
+                            ],
+                        },
+                        [MULTIPLE_OPERATORS_SERVICES_ATTRIBUTE]: [
+                            {
+                                nocCode: 'N1',
+                                services: ['service one', 'service two'],
+                            },
+                            {
+                                nocCode: 'N2',
+                                services: ['service one', 'service two'],
+                            },
+                            {
+                                nocCode: 'N3',
+                                services: ['service one', 'service two'],
+                            },
+                        ],
+                    },
+                });
+                const result = await getServerSideProps(ctx);
+                const expectedCheckedServiceList: ServicesInfo[] = mockServices.map(mockService => ({
+                    ...mockService,
+                    checked: false,
+                }));
+                expect(result.props.errors.length).toBe(0);
+                expect(result.props.serviceList).toEqual(expectedCheckedServiceList);
+                expect(result.props.buttonText).toEqual('Select All Services');
+                expect(result.props.operatorName).toBe('Test1' || 'Test2' || 'Test3');
+                expect(result.props.nocCode).toBe('N1' || 'N2' || 'N3');
+            });
+            it('should return expected props to the page when the page is visited by the user with errors on the session attribute', async () => {
+                const ctx = getMockContext({
+                    session: {
+                        [MULTIPLE_OPERATOR_ATTRIBUTE]: {
+                            selectedOperators: [
+                                { operatorPublicName: 'Test1', nocCode: 'N1' },
+                                { operatorPublicName: 'Test2', nocCode: 'N2' },
+                                { operatorPublicName: 'Test3', nocCode: 'N3' },
+                            ],
+                        },
+                        [MULTIPLE_OPERATORS_SERVICES_ATTRIBUTE]: {
+                            multiOperatorInfo: [
+                                {
+                                    nocCode: 'N1',
+                                    services: ['service one', 'service two'],
+                                },
+                            ],
+                            errors: mockError,
+                        },
+                    },
+                });
+                const result = await getServerSideProps(ctx);
+                expect(result.props.errors.length).toBe(1);
+                expect(result.props.errors[0]).toStrictEqual(mockError[0]);
```

Comment:
Does this expect not just want to be `expect(result.props.errors).toEqual(mockError)`? The expect on errors.length would also then be caught in one expect?

Reply:
Yeah good shout